packages/components/src/DateTimePickers/InputDateTimePicker/InputDateTimePicker.component.js

```diff
@@ -133,7 +133,11 @@ class InputDateTimePicker extends React.Component {
 			if (showPicker === isShown) {
 				return extra;
 			}
-			return { showPicker: isShown, ...extra };
+			return {
+				showPicker: isShown,
+				style: this.inputRef ? { width: this.inputRef.getBoundingClientRect().width } : {},
```

Comment:
Instead of getting the width of the input, requesting the DOM, can you just set a fixed size ?
I think you added that to fit the current behavior but a spec v2 will come soon, you can remove that.